eng/pipelines/templates/steps/analyze.yml

```diff
@@ -4,6 +4,7 @@ parameters:
   LintVersion: ''
   NonShipping: false
   LicenseCheck: true
+  IsSdkLibrary: true
```

Comment:
Should IsSdkLibrary and NonShipping be combined? They seem like they are representing something similar.